Maddori.Apple/Maddori.Apple/Screen/Home/Home/TeamManageViewController.swift

```diff
@@ -12,13 +12,11 @@ import SnapKit
 final class TeamManageViewController: BaseViewController {
     
     private var sections: [Section] = []
+    private lazy var teamCount = chageTeamView.teamDataDummy.count
     
     // MARK: - property
     
-    private let tempView: UIView = {
-        let view = UIView()
-        return view
-    }()
+    private let chageTeamView = ChangeTeamView()
```

Comment:
p4;
```suggestion
    private let changeTeamView = ChangeTeamView()
```
change...오타.. 수정...부탁드립니당

Reply:
진짜 '매'토리..

Reply:
[8649d40](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/319/commits/8649d407f55ba6514b16ea6cfa40e32ea4f55f28)

꾸벅 꾸벅 감사합니다!~